src/Expanders.jl

```diff
@@ -478,7 +478,7 @@ end
 function Selectors.runner(::Type{EvalBlocks}, x, page, doc)
     sandbox = Module(:EvalBlockSandbox)
     lines = Utilities.find_block_in_file(x.code, page.source)
-    cd(dirname(page.build)) do
+    cd(dirname(page.working_dir)) do
```

Comment:
Should probably remove `dirname` as `working_dir` should already be a directory? Necessary for `page.build` since it points to a file. This should hopefully fix the issues you were seeing.

Ditto for the other `runner`s.

Reply:
Will give it a shot!

Reply:
Did I accomplish what you were suggesting here?